### PR TITLE
getNbUnreadThreads

### PR DESCRIPTION
**Pull request by inalgnu**
this add a method to get the number of unread thread.

### CODE REVIEW

**Comment by smilesrg** [comment]
@zocimek Please reopen it!

File: EntityManager/MessageManager.php [hunk 1]
```diff
@@ -79,6 +79,36 @@ public function getNbUnreadMessageByParticipant(ParticipantInterface $participan
...
+     * Tells how many unread threads this participant has
+     *
+     * @param ParticipantInterface $participant
+     * @return int the number of unread threads
+     */
+    public function getNbUnreadThreadByParticipant(ParticipantInterface $participant)
+    {
+        $builder = $this->repository->createQueryBuilder('m');
+
+        return count($builder
```
Comment (smilesrg) [inline EntityManager/MessageManager.php]: Why not use query count?
```->select('count(m)')```
...
```getQuery()->getSingleScalarResult();```

File: EntityManager/MessageManager.php [hunk 2]
```diff
@@ -79,6 +79,36 @@ public function getNbUnreadMessageByParticipant(ParticipantInterface $participan
     }
 
     /**
+     * Tells how many unread threads this participant has
+     *
+     * @param ParticipantInterface $participant
+     * @return int the number of unread threads
+     */
+    public function getNbUnreadThreadByParticipant(ParticipantInterface $participant)
```
Comment (smilesrg) [inline EntityManager/MessageManager.php]: It's better to move this method to ThreadManager

**Comment by tgalopin** [comment]
@inalgnu, do you plan to finish this?

**Comment by tgalopin** [comment]
I close this as it's quite old, but if someone is interested in working on this, I can reopen it.

**Pull request closed without merging**